### PR TITLE
DOMA-8614 News Item Sharing getRecipients proxy function

### PR DESCRIPTION
**Pull request by toplenboren**
So, we need a small proxy function to call News Sharing Miniapp's getRecipients function

### CODE REVIEW

**Review by VKislov — APPROVED** [review]

File: apps/condo/domains/news/access/GetNewsSharingRecipientsService.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    if (user.deletedAt) return false
+    if (user.isAdmin || user.isSupport) return true
+
+    const b2bContext = await getById('B2BAppContext', b2bAppContextId)
+    if (!b2bContext) { return false }
+
+    const organizationId = get(b2bContext, 'organization', null)
+    if (!organizationId) { return false }
+
+    return await checkPermissionInUserOrganizationOrRelatedOrganization(user.id, organizationId, 'canReadNewsItems')
```
Comment (AleX83Xpert) [inline apps/condo/domains/news/access/GetNewsSharingRecipientsService.js:22]: Why `canReadNewsItems`?

File: apps/condo/index.js [hunk 2]
```diff
@@ -23,6 +23,7 @@ const { getWebhookModels } = require('@open-condo/webhooks/schema')
 const { PaymentLinkMiddleware } = require('@condo/domains/acquiring/PaymentLinkMiddleware')
 const FileAdapter = require('@condo/domains/common/utils/fileAdapter')
 const { VersioningMiddleware } = require('@condo/domains/common/utils/VersioningMiddleware')
+const { NewsSharingTestingApp } = require('@condo/domains/news/utils/testSchema/NewsSharingTestingApp')
```
[the remark this answers is not in the document]
Comment (toplenboren) [reply apps/condo/index.js:26]: Yep, thanks! This import shouldn't be here

File: apps/condo/domains/news/access/GetNewsSharingRecipientsService.js (hunk 1 above)
Comment (toplenboren) [reply apps/condo/domains/news/access/GetNewsSharingRecipientsService.js:22]: It was made intentionally. If you can read and manage newsItems, then you can send news to telegram and other services

File: apps/condo/domains/news/schema/GetNewsSharingRecipientsService.test.js [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+} = require('@condo/domains/miniapp/utils/testSchema')
+const { ERRORS } = require('@condo/domains/news/schema/GetNewsSharingRecipientsService')
+const { getNewsSharingRecipientsByTestClient } = require('@condo/domains/news/utils/testSchema')
+const { createTestOrganization } = require('@condo/domains/organization/utils/testSchema')
+
+
+const {
+    createTestOrganizationEmployeeRole,
+    createTestOrganizationEmployee,
+} = require('../../organization/utils/testSchema')
```
[the remark this answers is not in the document]
Comment (toplenboren) [reply apps/condo/domains/news/schema/GetNewsSharingRecipientsService.test.js:21]: Thanks, will fix!

File: apps/condo/domains/news/schema/GetNewsSharingRecipientsService.js [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+
+const GetNewsSharingRecipientsService = new GQLCustomSchema('GetNewsSharingRecipientsService', {
+    types: [
+        {
+            access: true,
+            type: 'input GetNewsSharingRecipientsInput { dv: Int!, sender: JSON!, b2bAppContext: B2BAppContextWhereUniqueInput! }',
+        },
+        {
+            access: true,
+            type: 'type GetNewsSharingRecipientsOutput { id: String!, name: String!, recipients: Int }',
```
[the remark this answers is not in the document]
Comment (toplenboren) [reply apps/condo/domains/news/schema/GetNewsSharingRecipientsService.js:99]: Thanks, will change to recipientsCount (this reallly is just the number of subscribers :-) )

File: apps/condo/domains/news/access/GetNewsSharingRecipientsService.js (hunk 1 above)
Comment (AleX83Xpert) [reply apps/condo/domains/news/access/GetNewsSharingRecipientsService.js:22]: > ...you can read and manage newsItems, then you can send news to telegram and other services

What about to check `canManage` permission?

Comment (toplenboren) [reply apps/condo/domains/news/access/GetNewsSharingRecipientsService.js:22]: will double check it

Comment (toplenboren) [reply apps/condo/domains/news/access/GetNewsSharingRecipientsService.js:22]: Got it, changed permission to canManageNewsItems. Thanks!

File: apps/condo/domains/news/schema/GetNewsSharingRecipientsService.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (toplenboren) [reply apps/condo/domains/news/schema/GetNewsSharingRecipientsService.js:99]: Done!

**Pull request merged**